File: Array & Hashing/1. Two Sum/two_sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashmap = {}

        for idx, ele in enumerate(nums):
          complement = target - ele
          if (complement in hashmap):
              return [idx, hashmap[complement]]
          hashmap[ele] = idx

File: Array & Hashing/1. Two Sum/twoSum.ts
const twoSum = function(nums: number[], target: number): number[] {
  const hashmap: { [key: number]: number } = {};
  
  for (const [index, element] of nums.entries()) {
    const complement = target - element;
    if (complement in hashmap) {
      return [index, hashmap[complement]];
    }
    hashmap[element] = index;
  }
};